### DISCUSSION TITLE
how to do pdf masking or text manipulation in pdf ?

### DISCUSSION THREAD

**Discussion by Laxmi530** [jsvine/pdfplumber#880] [Ask for help with specific PDFs]
Hi,
I have a multi-page pdf wanted to mask certain fields in the pdf file, suppose like account number, phone number and name or wanted to replace other text in that place. 

Can someone please guide me how to do this.
Thanking you in advance.

**Comment by jsvine** [comment]
Hi @Laxmi530. Unfortunately, `pdfplumber` does not write/create/edit PDFs. You may be interested in the libraries mentioned here: https://github.com/jsvine/pdfplumber/discussions/838#discussioncomment-5292149

**Reply by Laxmi530** [reply]
Okay, Thanks @jsvine .
